src/Cardano/Shell/Configuration/Lib.hs

```diff
@@ -89,16 +89,31 @@ finaliseCardanoConfiguration PartialCardanoConfiguration{..} = do
     finaliseCore :: PartialCore -> Either Text Core
     finaliseCore PartialCore{..} = do
 
-        coGenesisFile               <- lastToEither "Unspecified coGenesisFile"             pcoGenesisFile
-        coGenesisHash               <- lastToEither "Unspecified coGenesisHash"             pcoGenesisHash
+        coGenesisFile                   <- lastToEither "Unspecified coGenesisFile"
+                                            pcoGenesisFile
 
-        coStaticKeySigningKeyFile   <- lastToEither "Unspecified coStaticKeySigningKeyFile" pcoStaticKeySigningKeyFile
-        coStaticKeyDlgCertFile      <- lastToEither "Unspecified coStaticKeyDlgCertFile"    pcoStaticKeyDlgCertFile
+        coGenesisHash                   <- lastToEither "Unspecified coGenesisHash"
+                                            pcoGenesisHash
 
-        coRequiresNetworkMagic      <- lastToEither "Unspecified coRequiresNetworkMagic"    pcoRequiresNetworkMagic
-        coDBSerializeVersion        <- lastToEither "Unspecified coDBSerializeVersion"      pcoDBSerializeVersion
+        coNodeId                        <- lastToEither "Unspecified coNodeId"
+                                            pcoNodeId
 
-        coPBftSigThd                <- lastToEither "Unspecified coPBftSigThd"              pcoPBftSigThd
+        coNumCoreNodes                  <- lastToEither "Unspecified coNumCoreNodes"
+                                            pcoNumCoreNodes
+
+        coNodeProtocol                  <- lastToEither "Unspecified coNodeProtocol"
+                                            pcoNodeProtocol
+
+        let coStaticKeySigningKeyFile   = getLast pcoStaticKeySigningKeyFile
+        let coStaticKeyDlgCertFile      = getLast pcoStaticKeyDlgCertFile
+
+        coRequiresNetworkMagic          <- lastToEither "Unspecified coRequiresNetworkMagic"
+                                            pcoRequiresNetworkMagic
+
+        coDBSerializeVersion            <- lastToEither "Unspecified coDBSerializeVersion"
+                                            pcoDBSerializeVersion
```

Comment:
I don't think the new node makes us of such a param. You can double check, but I think we'll be able to drop this.